mlrun/runtimes/base.py

```diff
@@ -310,15 +310,19 @@ def run(
         local_code_path=None,
         auto_build=None,
         param_file_secrets: Dict[str, str] = None,
+        returns: List[Union[str, Dict[str, str]]] = None,
```

Comment:
```suggestion
        returns: Optional[List[Union[str, Dict[str, str]]]] = None,
```

Reply:
It is kind of obvious that it is optional as it is ` = None` so I don't like using the `typing.Optional` hint, but I'll add it from now on.

Reply:
also, check `mypy` and most IDEs (PyCharm and VSCode), they all assume Optional when `None` is assigned by default.